lib/dams/dams_object.rb

```diff
@@ -597,6 +597,22 @@ def to_solr (solr_doc = {})
 	      insertFacets solr_doc, "subject_topic", load_series(component.series)
 	      insertFacets solr_doc, "subject_topic", load_cruises(component.cruise)
 	      	      	
+          # uncomment for subject anatomy
+          #insertFacets solr_doc, "subject_anatomy", load_anatomys(component.anatomy)
+          #insertFacets solr_doc, "component_#{cid}_subject_anatomy", load_anatomys(component.anatomy)
+          insertFacets solr_doc, "subject_common_name", load_commonNames(component.commonName)
+          insertFacets solr_doc, "component_#{cid}_subject_common_name", load_commonNames(component.commonName)
```

Comment:
Why do components need a separate insertFacets call? aren't component facet values loaded into the primary facet such as subject_common_name? I'm unclear why we need the component_cid_XXX versions.